.github/workflows/test.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: ["3.7", "3.11"] # Empty string will trigger a build with the latest python version
+        python-version: ["3.7", "3.11". "3.12"] # Empty string will trigger a build with the latest python version
```

Comment:
Should we drop 3.11 here? I think we only want the lowest and the highest version.